lib/configure.js

```diff
@@ -355,9 +355,16 @@ function readFile(file) {
 
 
 function version() {
-    var fname = __dirname + '/../package.json';
-    var pkg = fs.readFileSync(fname, 'utf8');
-    return (JSON.parse(pkg).version);
+    //
+    // The last package.json version of manta before the manta v2 split was
+    // 1.0.2. This used to use the package.json version and clients are
+    // depending on ~1.0. Since Manta wanted to go to v2, we need to hardcode
+    // this so that old clients still work even as functionality is deprecated
+    // in Manta v2. The functionality that *does* still exist, is still
+    // compatible with the old tools and is likely to remain so. If not, this
+    // will need to be bumped too.
+    //
+    return ('1.0.3');
```

Comment:
Can we rename this apiVersion to explicitly differentiate from the package.json version?

Also can we set it to `'1.0.0'` rather than with the '.3'?  My only real justifications are:

- 1.0.0 satisfies the known clients
- the best practice I've tried to preach (and used in cloudapi and imgapi) is to set restify server.versions to just "X.0.0" versions, i.e. to attempt to indicate API versioning should only be on the major version

Reply:
I can rename the function, sure.

Reply:
I think 1.0.0 is weird since the version goes backward from the previous release. But if you want that, I don't care that strongly.

Reply:
Yes please. I don't think it is too plausible that a client was using `Accept-Version: >=1.0.2` and is going to be broken. I thnk having an API version on the restify endpoints of "1.0.3" is also weird.  I will happily take heat if this breaks someone.

Reply:
ok